pkg/kapp/cmd/app/delete.go

```diff
@@ -30,6 +30,11 @@ type DeleteOptions struct {
 	ResourceTypesFlags  ResourceTypesFlags
 }
 
+type changeFlags struct {
+	hasNoChanges     bool
+	isFullyDeleteApp bool
```

Comment:
instead of tracking isFullyDeleteApp, it's probably better to track whether changes were skipped. `SkippedChanges bool`

Reply:
Ok sure. So `SkippedChanges` flag will be false by default and I can make it true when 
```
if len(changes) != len(appliedChanges) {
	skippedChanges = true
}
```